### PR TITLE
Plugin refactor

### PR DESCRIPTION
**Pull request by AdrieanKhisbe**
Implement #137

### CODE REVIEW

**Comment by AdrieanKhisbe** [comment]
Feedback welcome @geek @mcdonnelldean @senecajs/core :)

**Comment by AdrieanKhisbe** [comment]
now takes care of #136, highlight 9 without metalsmith plugin

File: package.json [hunk 1]
```diff
@@ -33,14 +34,15 @@
     "metalsmith": "^1.7.0",
     "metalsmith-annotate": "^0.1.1",
     "metalsmith-download": "0.0.1",
+    "metalsmith-drafts": "0.0.1",
```
Comment (geek) [inline package.json]: you might be able to remove minimist too

File: package.json [hunk 2]
```diff
@@ -33,14 +34,15 @@
...
     "metalsmith-layouts": "^1.0.0",
     "metalsmith-less": "^2.0.0",
     "metalsmith-markdown": "^0.2.1",
-    "metalsmith-metallic": "^0.3.1",
     "metalsmith-move-up": "1.0.0",
     "metalsmith-register-partials": "^1.0.2",
     "metalsmith-serve": "0.0.4",
+    "metalsmith-start": "^1.4.2",
     "metalsmith-watch": "^0.2.0",
     "minimist": "^1.1.1"
```
Comment (geek) [inline package.json]: you can probably remove minimist

**Comment by AdrieanKhisbe** [comment]
Oops, I thought I did

**Comment by AdrieanKhisbe** [comment]
Et voilà :)

(@geek and all @senecajs/contributors happy new year :) :fireworks: )

**Comment by geek** [comment]
@AdrieanKhisbe Happy New Year!

**Comment by AdrieanKhisbe** [comment]
@geek  :smiley: 

(do you know how can we trigger deploy?
Thought it was automatic with surge)

**Comment by mcdonnelldean** [comment]
It's done by travis

https://github.com/senecajs/senecajs.org/blob/master/.travis.yml

**Comment by AdrieanKhisbe** [comment]
okey dokey!

**Comment by AdrieanKhisbe** [comment]
well, was broken...
cf https://travis-ci.org/senecajs/senecajs.org/builds/100376325

**Pull request merged**